src/components/scenes/GuiPluginListScene.tsx

```diff
@@ -341,6 +343,7 @@ class GuiPluginList extends React.PureComponent<Props, State> {
     // Filter disabled plugins:
     const activePlugins = bestOfPlugins(accountPlugins, accountReferral, undefined)
     plugins = plugins.filter(plugin => !activePlugins.disabled[plugin.pluginId])
+    plugins = plugins.filter(plugin => !disablePlugins[plugin.pluginId])
```

Comment:
Change to

`plugins = plugins.filter(plugin => !disablePlugins[plugin.id])`

`pluginId` is not specific enough as it's shared between different payment methods and also with plugins in the FiatPlugins framework.

Reply:
Replaced cleaner to give more control over pluginIds in list view and within plugins like creditcard